src/modules/Footer/components/Footer.js

```diff
@@ -28,6 +30,7 @@ class Footer extends Component {
             <p className="my-2">PL 29, 00023 VALTIONEUVOSTO</p>
             <p className="my-0">PUH. 029 533 0004, <a href="http://www.minedu.fi">WWW.MINEDU.FI</a></p>
           </div>
+          <Button variant="raised" color="primary">Jei</Button>
```

Comment:
Good example line but we can go forward without it.

Reply:
Hmm.. I removed it already... I need to check if I made push.

Reply:
It was removed. You are checking older version.